quiffen/core/qif.py

```diff
@@ -184,18 +184,26 @@ def parse(cls, filepath, separator='\n', day_first=True):
                 new_account = Account.from_string(section)
                 accounts[new_account.name] = new_account
                 last_account = new_account.name
-            elif '!Type' in header_line and not accounts:
+            elif '!Type:Invst' in header_line:
+                # Investment
+                new_investment = Investment.from_string(section, separator=separator, day_first=day_first,
+                                                        line_number=line_number)
+                accounts[last_account].add_transaction(new_investment, 'Invst')
+            elif not accounts and str.lower(header_line) in VALID_TRANSACTION_ACCOUNT_TYPES:
```

Comment:
Can we use `header_line.lower()` here rather than the `str` classmethod?

Reply:
Now changed. I also included the whitespace stripping that is performed in other `elif` cases using `header_line.lower()`.